cmake/dependencies.cmake

```diff
@@ -88,6 +88,12 @@ set_target_properties(boost PROPERTIES
     INTERFACE_LINK_LIBRARIES "${Boost_LIBRARIES}"
     )
 
+if(ENABLE_LIBS_PACKAGING)
+  foreach (library ${Boost_LIBRARIES})
+    add_install_step_for_lib(${library})
```

Comment:
Looks weird for me. What's the point of packing boost libs?
Explain plz

Reply:
Because when we build deb package, it does not install boost libraries, and on a fresh system it will lack boost (see #749 )

Reply:
Quite strange way of fixing. Maybe it's possible to set dependencies in the deb package itself?

Reply:
We need a specific version of boost that is used during build